ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml

```diff
@@ -22,6 +22,12 @@ images:
       - registry.ci.openshift.org/ci/maistra-builder:upstream-master
   to: src-test
 releases:
+  arm64-latest:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
```

Comment:
Do we need this `nightly` thing? Can we use something more stable?

Reply:
All the others teams using arm cluster have set the same configuration for their jobs, that is why I set like this. I'm going to see if we can use something like stable

Reply:
I added this configuration to test because all the team are working with `nightly`
```
  arm64-latest:
    release:
      architecture: arm64
      channel: stable
      version: "4.14"
```